style.css

```diff
@@ -43,21 +50,282 @@ body {
   flex-direction: column;
 }
 
-ul {
+.socials-mainlist {
   display: flex;
   padding: 0;
   margin-top: 10px;
 }
 
-ol {
+.socials-list {
   padding: 0;
   padding-right: 20px;
 }
 
-h4 {
+.letsconnect {
+  width: 160px;
+  height: 24px;
   color: #7f8cff;
+  font-size: 20px;
 }
 
 .logo {
   color: #6070ff;
 }
+
+.works {
+  display: grid;
+}
+
+.resume {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-style: normal;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  background-color: #fff;
+  border-radius: 15px 15px 15px;
+  padding: 0.75em;
+  width: 170px;
+  height: 50px;
+  margin-top: 5px;
+  margin-bottom: 30px;
+  border: 1px solid #6070ff;
+}
+
+#p-tag1 {
+  font-size: clamp(16px, 4vw, 20px);
+}
+
+#p-tag {
+  font-size: clamp(16px, 4vw, 20px);
+  margin: 0;
+}
+
+#h-tag {
+  font-size: clamp(32px, 4vw, 45px);
+  font-weight: 700;
+}
+
+#tonic2 {
+  top: 170px;
+}
+
+.tonic {
+  margin-bottom: 0;
+}
+
+.multi {
+  width: 250px;
+  height: 80px;
+  line-height: 40px;
+  margin-bottom: 0;
+}
+
+.work-cards {
+  display: flex;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-color: #fff;
+  gap: 12px;
+  padding: 10px;
+  position: relative;
+  left: 8px;
+  top: 150px;
+  margin-bottom: 20px;
+  margin-right: 15px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.canopy {
+  font-size: 13px;
+  font-weight: 600;
+  padding: 0;
+}
+
+.canopy-list {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  font-size: 13px;
+  padding: 0;
+  line-height: 16px;
+  margin-top: 10px;
+}
+
+.canopy-list > li {
+  color: #7a869a;
+  font-weight: 600;
+  font-size: 13px;
+}
+
+#h-tagaboutme {
+  font-size: clamp(40px, 4vw, 45px);
+  font-weight: 900;
+  width: 250px;
+  height: 101px;
+  line-height: 52px;
+}
+
+.aboutme {
+  display: flex;
+  flex-direction: column;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-color: #fff;
+  padding: 100px 5%;
+  border-radius: 0 100px 0 0;
+  position: relative;
+  left: 0;
+  top: 190px;
+}
+
+.about-header {
+  color: #172b4d;
+  width: 200px;
+  height: 80px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: justify;
+}
+
+.work-img {
+  flex: none;
+  flex-grow: 0;
+  border-top: 5px;
+  background-color: #fff;
+  width: 100%;
+}
+
+.small-langul {
+  color: #6070ff;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-style: normal;
+  padding: 4px 0;
+  position: relative;
+  margin-bottom: 5px;
+}
+
+.small-langli {
+  color: #6070ff;
+  background-color: #ebebff;
+  align-items: center;
+  font-style: normal;
+  padding: 4px 8px;
+  margin: 0 5px;
+  list-style-type: none;
+  border-radius: 8px;
+  font-size: 12px;
+  border: 0;
+}
+
+.seeproject.active:hover {
+  background-color: #000;
+}
```

Comment:
- You've done well implementing the hover states✨. However, you didn't quite get them right. The background color of the button should be `#6070ff` and the text color should be `#fff`
- Also, please add styling for when the button is pressed. You can do this using the `:active` pseudo selector. Please follow the design guide provided by figma
<img width="447" alt="image" src="https://user-images.githubusercontent.com/37116322/219853083-8aa57101-6a93-44a3-861a-0128118c6169.png">